ui/src/widgets/ui-gauge/types/UIGaugeDial.vue

```diff
@@ -93,7 +93,6 @@ export default {
     },
     watch: {
         value: function (val) {
-            this.resize()
```

Comment:
Removed this gauge resizing from the value watcher as it's not necessary to resize on each payload update and it's memory consuming.